app/models/glitch/keyword_mute.rb

```diff
@@ -33,68 +33,74 @@ def invalidate_cached_matchers
     Rails.cache.delete(TagMatcher.cache_key(account_id))
   end
 
-  class RegexpMatcher
+  class CachedKeywordMute
```

Comment:
There are two representations of a keyword mute:

- `Glitch::KeywordMute`, the model mapping the `keyword_mutes` table
- This class

I made it because it wasn't clear to me exactly what goes into a marshalled form of an ActiveRecord object.  Creating my own class is easier to understand and control, and it means that auxillary data that could be associated with a `Glitch::KeywordMute` (e.g. statistics) does not need to occupy space in Redis memory.